app/src/main/java/com/sopt/now/network/dto/ResponseFollowListDto.kt

```diff
@@ -0,0 +1,43 @@
+package com.sopt.now.network.dto
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseFollowListDto(
+    @SerialName("page")
+    val page:Int,
+    @SerialName("per_page")
+    val per_page:Int,
```

Comment:
안드로이드에서는 변수명을 카멜 케이스로 작성해주시는 것을 권장합니다!
서버에서 다르게 오는 경우를 대응해주기 위해서 @SerialName 어노테이션을 활용하는 건데요, 카멜케이스 형태의 변수에 저장하는 것을 추천드려요 ~~
```suggestion
    @SerialName("per_page")
    val perPage: Int,
```